tools/build.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+import argparse
+from setup import get_build_dir
+import subprocess
+import os
+import time
+
+project_root = get_build_dir() / '..'
```

Comment:
This would prevent the build directory from being moved further down in the levels later on, which was one idea of better organizing multiple versions on the decomp.

Instead, better pick `ROOT` from `setup_common`.